dev/delme_temp_vcftomaf.cwl

```diff
@@ -0,0 +1,68 @@
+cwlVersion: v1.0
```

Comment:
I don't currently understand the purpose of this workflow in the context of the INFO/AF thing, and the name seems to suggest it is not desired or needed in the long run. Is it supposed to be part of this PR?

Reply:
Oh, so, this was actually a patch wf that I intend to delete later. If not right, now, by the next major updates. I was afraid to lose it right away with all the concurrent things happening. I guess I've panic-added this. It can be ignored.